Windows Dev Build Failure :  error: unknown type name 'int32_t'

Clean build after clearing build directory.

```
Trying to build : Version string: 3.3.0+1128~gbb9b701fe


[ 14%] Building C object bin/CMakeFiles/lib_darktable.dir/common/grouping.c.obj
In file included from C:/msys64/home/chill/darktable/src/common/grouping.c:20:
C:/msys64/home/chill/darktable/src/common/grouping.h:24:45: error: unknown type name 'int32_t'
   24 | void dt_grouping_add_to_group(int group_id, int32_t image_id);
      |                                             ^~~~~~~
C:/msys64/home/chill/darktable/src/common/grouping.h:27:35: error: unknown type name 'int32_t'
   27 | int dt_grouping_remove_from_group(int32_t image_id);
      |                                   ^~~~~~~
C:/msys64/home/chill/darktable/src/common/grouping.h:30:39: error: unknown type name 'int32_t'
   30 | int dt_grouping_change_representative(int32_t image_id);
      |                                       ^~~~~~~
C:/msys64/home/chill/darktable/src/common/grouping.h:33:43: error: unknown type name 'int32_t'
   33 | GList *dt_grouping_get_group_images(const int32_t imgid);
      |                                           ^~~~~~~
C:/msys64/home/chill/darktable/src/common/grouping.c: In function 'dt_grouping_add_to_group':
C:/msys64/home/chill/darktable/src/common/grouping.c:30:3: error: implicit declaration of function 'dt_grouping_remove_from_group'; did you mean 'dt_grouping_add_to_group'? [-Werror=implicit-function-declaration]
   30 |   dt_grouping_remove_from_group(image_id);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |   dt_grouping_add_to_group
cc1.exe: all warnings being treated as errors
make[2]: *** [bin/CMakeFiles/lib_darktable.dir/build.make:427: bin/CMakeFiles/lib_darktable.dir/common/grouping.c.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:2336: bin/CMakeFiles/lib_darktable.dir/all] Error 2
make: *** [Makefile:172: all] Error 2

```

Probably related to  @jenshannoschwalm change on https://github.com/darktable-org/darktable/pull/6549

--- a/src/common/grouping.h
+++ b/src/common/grouping.h
@@ -19,6 +19,7 @@
 #pragma once
 
 #include <glib.h>
+#include <stdint.h>
 
 /** add an image to a group */
 void dt_grouping_add_to_group(int group_id, int32_t image_id);

